express-notes-api/server.js

```diff
@@ -0,0 +1,139 @@
+import express from 'express';
+import { readFile, writeFile } from 'node:fs/promises';
+
+const app = express();
+
+app.use(express.json());
+
+// gets a list of notes in an array
+app.get('/api/notes', async (req, res) => {
+  try {
+    const data = await fetchFile();
+    const array = [];
+
+    for (const note in data.notes) {
+      array.push(data.notes[note]);
+    }
+    res.status(200).json(array);
```

Comment:
It's good to explicitly set the status code as you've done, but I will mention that by default,  express sets it to 200.